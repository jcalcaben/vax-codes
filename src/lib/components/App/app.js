import React from "react";

import Header from "../Header";
import Main from "../Main";
import Footer from "../Footer";

const App = (props) => {
  const { children } = props;

  return (
    <>
      <Header />
      <Main>{children}</Main>
      <Footer />
    </>
  );
};

export default App;
