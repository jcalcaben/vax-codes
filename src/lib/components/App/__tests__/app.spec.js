import React from "react";
import App from "../app";
import ShallowRenderer from 'react-test-renderer/shallow';

const renderer = new ShallowRenderer();

describe("App component", () => {
  test("renders the correct basic shape", () => {
    renderer.render(<App>Content</App>);
		const output = renderer.getRenderOutput();

    expect(output).toMatchSnapshot();
  });
});
