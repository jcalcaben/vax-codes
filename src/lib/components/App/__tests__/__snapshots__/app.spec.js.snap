// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App component renders the correct basic shape 1`] = `
<React.Fragment>
  <Header />
  <Main>
    Content
  </Main>
  <Footer />
</React.Fragment>
`;
